components/kubearchive/production/base/database-secret.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: database-secret
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: '-1'
+spec:
+  dataFrom:
+    - extract:
+        key: production/kubearchive/rds
```

Comment:
Could you please note with a comment or something that this is just a placeholder being modified on each cluster installation?

Reply:
Added